terraform/modules/gcf_gen2_pubsub_source_repo/variables.tf

```diff
@@ -1,78 +1,80 @@
 variable "name" {
   description = "Name of the function."
-  type = string
+  type        = string
 }
 
 variable "description" {
   description = "Description of the pub/sub function."
-  type = string
+  type        = string
 }
 
 variable "region" {
   description = "Region where function is living"
-  default = "europe-west1"
-}
-
-variable "function_region" {
-  description = "Region where function is living"
-  default = "europe-west1"
+  default     = "europe-west1"
 }
 
 variable "runtime" {
   description = "Runtime where function is operating."
-  type = string
-  default = "python310"
+  type        = string
+  default     = "python310"
```

Comment:
Do we want to change this default?

Reply:
I forgot about this point, so thanks for calling it out!